src/Render.php

```diff
@@ -106,7 +106,12 @@ public function fetchCachedRequest()
                 // maximum duration, otherwise a proxy/cache might keep the
                 // cache twice as long in the worst case scenario, and now it's
                 // only 50% max, but likely less
-                $response->setSharedMaxAge($this->cacheDuration() / 2);
+                // 's_maxage' sets the cache for shared caches.
+                // max_age sets it for regular browser caches
+                $response->setCache(array(
+                    'max_age'       => $this->cacheDuration() / 2,
+                    's_maxage'      => $this->cacheDuration() / 2
+                ));
             }
```

Comment:
Could of points:

* master (to be v3) is PHP 5.4+ and we've switched to sort array syntax
* We're also sticking with fluent interfaces for things like this… can you switch to something like:
```php


                $age = $this->cacheDuration() / 2;
                $response->setMaxAge($age)->setSharedMaxAge($age);
```